api/app/__init__.py

```diff
@@ -27,15 +27,41 @@
 redis = FlaskRedis(app)
 LOGGER = Logger().get_logger()
 
-
-
 import routes
 
 migrate = Migrate(app, db)
 
 manager = Manager(app)
 manager.add_command('db', MigrateCommand)
 
+from organisation.resolver import OrganisationResolver
+
+def get_domain():
+    # TODO: Remove this test-related hack!
+    if app.config['TESTING'] and 'HTTP_ORIGIN' not in request.environ and 'HTTP_REFERER' not in request.environ:
```

Comment:
Very hacky solution. @KaleabTessera @jaderabbit @amritpurshotam - let me know if I'm maybe missing some obvious better solution? Otherwise will submit this PR in the interests of time and create an issue to improve this